mtz-update-password.html

```diff
@@ -336,6 +368,18 @@
         return Boolean(value);
       },
       /**
+       * Sets invalid on this element when any of the errors are set in the validationErrors object.
+       * @private
+       *
+       * @param {Object} errors
```

Comment:
This parameter name needs to be updated to reflect the parameter name in the function